cds_ils/importer/documents/importer.py

```diff
@@ -251,6 +252,10 @@ def search_for_matching_documents(self):
             results = search.scan()
             matches += [x.pid for x in results if x.pid not in matches]
 
+            for match in matches:
+                document = document_class.get_record_by_pid(match)
+                if document.get("edition") != self.json_data.get("edition"):
```

Comment:
if one have does not have the `edition` field, and the other has it, it will pass the test. Is it the expected behaviour?

Reply:
yes, thats exactly what we want